02-molecules/cta/Cta.component.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../01-atoms/buttons/Button.component';
+
+const withModifiers = (className) => (modifiers) =>
+  `${className} ${modifiers.map((mod) => ` ${className}--${mod}`)}`;
```

Comment:
This is the second time I've seen this pattern. The first is in the Buttons component. Can we move this to a common set of utility functions that may be used by many components?

Reply:
Although I guess this is not technically the same thing. For Buttons the modifier is optional. But for this CTA component, the modifiers are required.

Reply:
Final note: I see that the cta_modifiers have a default value of an empty string. This will create the class "cta " with the little dangling space at the end.